org.kde.kamoso.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "org.kde.kamoso",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kamoso",
+    "rename-icon": "kamoso",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=host",
```

Comment:
why 'host'

Reply:
This part definitely needs testing indeed.

Reply:
Replaced that with video & pictures only